rpms/microshift.yml

```diff
@@ -0,0 +1,22 @@
+# Disable automated build because it is intended to be built after a release is promoted.
+mode: disabled
+content:
+  source:
+    git:
+      branch:
+        target: main
+        # [lmeyer 2022-07-10] main branch is indeed for 4.10 right now and that is where development is occurring.
+        # We will want this to be release-4.y branches managed by Test Platform once work is beginning on
+        # other releases but that would just get in the way of dev for the current state of affairs.
+      url: git@github.com:openshift/microshift.git
+    specfile: packaging/rpm/microshift.spec
+    modifications:
+    - action: command
+      command: microshift-rebase
+      env:
+        RELEASE_NAME: "{release_name}"
```

Comment:
@vfreex what is supposed to set the `release_name` var so that formatting this string works?

Reply:
Something like `4.12.3`, `4.12.0-rc.4`. Doozer sets this var internally: https://github.com/openshift/doozer/pull/654/files#diff-814875ff0d43b12d4d3bcf2489cb6d7c5d04e72aedd08390042ebda2d2c8ddb3R119